content/source/docs/enterprise/install/encryption-password.html.md

```diff
@@ -5,27 +5,26 @@ page_title: "Encryption Password - Install and Config - Terraform Enterprise"
 
 # Terraform Enterprise Encryption Password
 
-During the installation of Terraform Enterprise,
-a password is used to encrypt sensitive information at
-rest. The default value is auto-generated, but we
-strongly suggest you create your own password.
+When using internally-managed Vault, Terraform Enterprise requires that the operator specify a password that will be
```

Comment:
```suggestion
When using internally managed Vault, Terraform Enterprise requires that the operator specify a password that will be
```

Minor suggestion - I don't think we need this to be hyphenated, but I'm open to being wrong.

Reply:
I personally don't have a preference. I started this pull request without the hyphen and changed to the hyphen from another review.

Reply:
Thinking about it a bit more, I think there's some precedent for omitting the hyphen. Specifically, we don't use "externally-managed" when referring to [external Vault](https://www.terraform.io/docs/enterprise/before-installing/vault.html). The only other place I could find that we use the phrase "internally managed" when referring to Vault is in the [VMWare reference architecture](https://www.terraform.io/docs/enterprise/before-installing/reference-architecture/vmware.html#infrastructure-requirements) and it doesn't use the hyphen.

Still not wildly opinionated on it, but was trying to make sure we were consistent.

Reply:
Speaking as the original reviewer- lets be consistent, the hyphen was a pedantic (and opinionated) grammatical switch. Apologies for the noise, I should have looked at the docs as a whole before asking for a pedantic change.